.circleci/config.yml

```diff
@@ -1,12 +1,67 @@
 version: 2.1
 
-jobs:
-  build:
+commands:
+  get_test_results_path:
+    parameters:
+      path:
+        type: string
+        default: /tmp/test-results
+    steps:
+      - run: echo << parameters.path >>
+  store_results:
+    parameters:
+      path:
+        type: string
+        default: /tmp/test-results
+    steps:
+      - store_test_results:
+          path: << parameters.path >>
+      - store_artifacts:
+          path: << parameters.path >>
+
+executors:
+  go:
     docker:
       - image: circleci/golang:1.13.3
+    shell: /usr/bin/env bash -euo pipefail -c
+
+jobs:
+  go-test:
+    executor: go
     steps:
+      - checkout
+      - run: mkdir -p get_test_results_path
```

Comment:
Sorry, I think my comment about this was a bit unclear. You can only reference commands as `steps`, they don't get expanded in places like this, so this just results in the literal string `get_test_results_path`.